lib/usermanager.js

```diff
@@ -53,6 +52,22 @@ exports = module.exports = {
     'DuplicateUserError': DuplicateUserError
   },
 
+  /**
+   * checks if a user is permitted the action on the resource
+   *
+   * @param {string} action
+   * @param {objectid} userId
+   * @param {objectid} tenantId
+   * @param {object} a content item
+   * @param {callback} next (function (err, isAllowed))
+   */
+
+  hasPermission: function(action, user, contentItem, next) {
+    permissions = require('./permissions');
+    var resource = permissions.buildResourceString(user.tenant._id, '/api/user/resetpassword/' + contentItem);
+    permissions.hasPermission(user._id, action, resource, next);
```

Comment:
I don't think the `contentItem` parameter is correct here if you're only looking to restrict access to the `/api/user/resetpassword` endpoint.